docs/src/routes/docs/components/radio-group.svx

```diff
@@ -0,0 +1,92 @@
+---
+name: RadioGroup
+relatedComponents:
+  - name: RadioButton
+    link: ./docs/components/radio-button
+---
+
+<script>
+  import { RadioGroup } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  const items = [
+    { value: '1', label: 'one' },
+    { value: '2', label: 'two' },
+  ];
+
+  const colors = [
+    { value: '#000000' },
+    { value: '#ff0000' },
+    { value: '#00ff00' },
+    { value: '#ffffff' },
+  ];
+</script>
+
+# RadioGroup
+
+A group of radio buttons to present a list of selectable options.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <RadioGroup {items} name="numbers" />
+    <RadioGroup color items={colors} name="colors" />
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  const items = [
+    { value: '1', label: 'one' },
+    { value: '2', label: 'two' },
+  ];
+
+  const colors = [
+    { value: '#000000' },
+    { value: '#ff0000' },
+    { value: '#00ff00' },
+    { value: '#ffffff' },
+  ];
+</script>
+
+<RadioGroup {items} name="numbers" />
+<RadioGroup color items={colors} name="colors" />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`items`**==&nbsp;<span class="required">*</span> | | `Array` | An array of `{ value, label, disabled }` objects, where only the value is required. Must contain at least one value. |
+| ==**`name`**==&nbsp;<span class="required">*</span> | | `string` | The name to assign to all radiobuttons belonging to the same group. Check [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname) for more information. |
+| **`value`** | `null` | `any` | The currently selected value of the group. |
+| **`labelsLeft`** | `false` | `boolean` | Places the labels to the left of the radiobuttons. |
```

Comment:
Doesn't this affect only the style? Based on what do you categorize props as functional or style?

Reply:
I honestly don't know. But I do agree that this is more of a style prop